src/Chapter2.hs

```diff
@@ -349,7 +363,10 @@ Implement a function that returns only the first half of a given list.
 "b"
 -}
 -- PUT THE FUNCTION TYPE IN HERE
-firstHalf l = error "firstHalf: Not implemented!"
+
+firstHalf :: [a] -> [a]
+firstHalf [] = []
```

Comment:
Actually, this case is included in the next line solution too, so can be safely removed ✨